apps/passport-client/components/shared/DownloadingPCDs.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+import { RippleLoader } from "../core/RippleLoader";
+
+export function DownloadingPCDs() {
```

Comment:
`DownloadingPCDs` and `SyncingPCDs` are the same

replace with a single component ?